README.md

```diff
@@ -13,38 +12,54 @@ Documentation: https://help.syncfusion.com/maui/Linear Gauge/getting-started
 * macOS 10.15 or higher, using Mac Catalyst.
 * Windows 11 and Windows 10 version 1809 or higher, using [Windows UI Library (WinUI) 3](https://learn.microsoft.com/en-us/windows/apps/winui/winui3/).
 
-## Requirements to run the sample
+To run this application, you need to first clone the getting-started-with-the-dotnet-maui-linear-gauge repository and then open it in Visual Studio 2022. Now, simply build and run your project to view the output.
 
-* [Visual Studio 2022 Preview](https://learn.microsoft.com/en-us/visualstudio/releases/2022/release-notes-preview) version 17.6.0 (.NET MAUI version 6.0.552) with .NET 6.0 and .NET 7.0
-* [Microsoft Visual Studio 2022](https://learn.microsoft.com/en-us/visualstudio/releases/2022/release-notes) version 17.5.1 with (.NET MAUI version 6.0.552) with .NET 6.0 and .NET 7.0
-* [Visual Studio 2022 for Mac Preview](https://visualstudio.microsoft.com/vs/mac/preview/) version 17.5 (.NET MAUI version 6.0.552) with .NET 6.0 and .NET 7.0
```

Comment:
Why is this section deleted?..please check carefully all the deleted sections

Reply:
Reverted